README.md

```diff
@@ -298,6 +298,12 @@ To do the same for the 10h16 skin, run the following Docker command correspondin
 
 If you want to debug problems in the Redux data flow, add the parameter `-e REDUX_LOG=on` to the command line above
 
+### Automatic recompilation of assets during development (wmde19)
+
+If you are working on the JavaScript files of the wmde19 skin and need automatic recompilation when a files changes, add the following setting to your `config.dev.json` file:
+
+	"assets-path": "http://localhostz:7072" 
```

Comment:
Is this `8082 minus 1010` based on some actual convention or is it just a joke? I'm curious 😄

Reply:
For all the FUN web applications I'm trying to use ports in the 808x range (8082 for FundraisingFrontend, 8083 for the Fundraising, 8084 for Banners) for the PHP side of things. And since I liked the repeating 8, I chose a repeating 7. It makes sense in my mind, but I fear I'm alone here :neckbeard: 🤓